Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioRecordViewController.swift

```diff
@@ -179,80 +178,79 @@ private let zmLog = ZMSLog(tag: "UI")
-            topContainer.top == view.top
-            topContainer.right == view.right
-            topContainer.bottom == bottomContainer.top
-        }
-        
-        constrain(topContainerView, topTooltipLabel, buttonOverlay) { topContainer, topTooltip, overlay in
-            topContainer.centerY == topTooltip.centerY
-            topTooltip.right == overlay.left - 12
-        }
-        
-        constrain(bottomContainerView, buttonOverlay, topSeparator) { container, overlay, separator in
-            separator.height == .hairline
-            separator.right == overlay.left - 8
-            separator.left == container.left + 16
-            separator.top == container.top
-        }
-        
-        self.recordingDotViewHidden = constrain(bottomContainerView, timeLabel) { container, timeLabel in
-            timeLabel.centerY == container.centerY
-            timeLabel.left == container.left + margin
-        }
-        
-        self.recordingDotViewHidden?.active = false
-        
-        self.recordingDotViewVisible = constrain(bottomContainerView, timeLabel, recordingDotView) { container, timeLabel, recordingDotView in
-            
-            timeLabel.centerY == container.centerY
-            timeLabel.left == recordingDotView.right + 24
-            
-            recordingDotView.width == recordingDotView.height
-            recordingDotView.width == 8
-            
-            recordingDotView.left == container.left + margin + 8
-            recordingDotView.centerY == container.centerY
-        }
-        self.recordingDotViewVisible?.active = true
-        
-        
-        constrain(bottomContainerView, buttonOverlay, rightSeparator) { container, overlay, rightSeparator in
-            rightSeparator.right == container.right
-            rightSeparator.left == overlay.right + 8
-            rightSeparator.top == container.top
-            rightSeparator.height == .hairline
-        }
-        
-        constrain(bottomContainerView, timeLabel, audioPreviewView, cancelButton, buttonOverlay) { container, timeLabel, previewView, cancelButton, overlay in
-            previewView.left == timeLabel.right + 8
-            previewView.top == container.top + 12
-            previewView.bottom == container.bottom - 12
-            previewView.right == overlay.left - 12
-
-            cancelButton.centerY == container.centerY
-            cancelButton.right == container.right
-            cancelButton.width == cancelButton.height
-            cancelButton.width == 56
-            overlay.right == cancelButton.left - 12
-        }
+        [bottomContainerView,
+         topContainerView,
+         button,
+         topTooltipLabel,
+         buttonOverlay,
+         topSeparator,
+         timeLabel,
+         recordingDotView,
+         audioPreviewView,
+         cancelButton,
+         rightSeparator].forEach(){ $0.translatesAutoresizingMaskIntoConstraints = false }
+
+        var constraints = [NSLayoutConstraint]()
```

Comment:
Using the array literal is usually more idiomatic in Swift.

```suggestion
        var constraints: [NSLayoutConstraint] = []
```

Reply:
Also, why are you not activating all constraints at once with `NSLayoutConstraint.activate`?